snappybouncer/models.py

```diff
@@ -79,21 +81,20 @@ class Ticket(models.Model):
     def __unicode__(self):
         return "%s" % self.message
 
-# Make sure new tickets are sent to Snappy via Celery
-from django.db.models.signals import post_save
-from django.dispatch import receiver
-
 
+# Make sure new tickets are sent to Snappy via Celery
 @receiver(post_save, sender=Ticket)
-def fire_snappy_if_new(sender, instance, created, **kwargs):
-    from snappybouncer.tasks import create_snappy_ticket
+def relay_to_helpdesk(sender, instance, created, **kwargs):
+    from snappybouncer.tasks import create_snappy_ticket, create_casepro_ticket
```

Comment:
apologies this rename from `fire_snappy_if_new` to `relay_to_helpdesk` litters a lot of this PR.